src/analyzer.ts

```diff
@@ -261,7 +273,7 @@ export class Analyzer {
       warnings.push({
         code: 'could-not-load',
         message: `Unable to load import: ${error.message || error}`,
-        sourceRange: scannedImport.sourceRange,
+        sourceRange: scannedImport.urlSourceRange || scannedImport.sourceRange,
```

Comment:
If we had strict null checks I wouldn't worry about it, but I haven't had a chance to fix the remaining issues so I fall back to doing paranoid checks like this.